protos/action/action.proto

```diff
@@ -166,6 +172,14 @@ message GotoLocationResponse {
     ActionResult action_result = 1;
 }
 
+message SetActuatorRequest {
+  int32 index = 1; // Index of actuator (starting with 1)
```

Comment:
I assume that's 1-indexed because that's how the MAVLink message is specced?

Reply:
Yep. Is this a problem?

Reply:
No, I guess it's ok.